mqtt/src/test/java/akka/stream/alpakka/mqtt/javadsl/MqttSourceTest.java

```diff
@@ -45,12 +47,19 @@
 
   @BeforeClass
   public static void setup() throws Exception {
+    serverPort = AvailablePort$.MODULE$.nextPort();
+    serverWebSocketPort = AvailablePort$.MODULE$.nextPort();
```

Comment:
This is probably a left over. `AvailablePort.nextPort();` should work fine here.